docs/docs/installation.md

```diff
@@ -6,11 +6,11 @@ title: Installation
 
 # Installation
 
-## Via package managers
+## Using a package manager
 
-### `homebrew` (MacOS and Linux)
+### Homebrew
 
-```bash
+```bash tab={"label":"Homebrew"}
```

Comment:
@kaplas Just wondering, would you expect it to render a single tab?

I am the author of the plugin (; A single tab was not implemented, because I had no real use case around. Would it make sense? Very easy to make this work.

Reply:
@mrazauskas Nice that you reached out! I wouldn't say that there really is a good use case for a feature like that. Eg. here in our case we didn't have any other package managers yet, but I just wanted to "future-proof" that part of the documentation already at this point, and thus I added the tabbing info there as well.

I implemented winget releases today, so our single-tab use case might be useless already tomorrow.

That said, I was confused though that it didn't render just a single tab in this case. So even though the feature is not so much useful, I did expect it to behave differently than what it did. You can decide yourself whether this is a good enough reason to implement the single-tab feature 😅

Thanx for your plugin, once again!